src/components/Header/index.js

```diff
@@ -0,0 +1,28 @@
+import "./header.scss";
+const Header = ({ actionType }) => {
+  return (
+    <header>
+      <div id="logo">
+        <img src="./../logo-pic.svg" alt="logo" className="resp-img" />
+      </div>
+      {actionType === "login" && (
```

Comment:
combine the elements into one block by condition. and if you have a radically different look, isn't it easier to just make 2 different components?